### DISCUSSION TITLE
Unable to create a project in Agama lab

### DISCUSSION THREAD

**Discussion by toroarnaud** [JanssenProject/jans#6548] [Q&A] [answered]
Hi,

It's my first time using the Agama lab. I've followed the instructions on the landing page that is:

1. Install Agama-Lab GitHub application: The Agama-Lab is installed with the permissions to access all my repositories
2. Select repository: This is what does not work for me. When trying the selection for "Your repositories" the combo box shows "Loading..." continuously and my repository never appears. If I enter the URL of the repo manually, when clicking on "Submit", the following message is displayed:
"_Failed to initialize git repo! Please check, You must have at least one commit._"
However I do have a README.md file commited in the repository.
I don't know what's wrong and what else to try... I can't wait to try this lab... Any help would greatly be appreciated!

**Comment by nynymike** [comment]
@SafinWasi any ideas?

**Reply by SafinWasi** [reply]
Hi @toroarnaud 
Please follow the following steps for troubleshooting:
1. On your browser, open the network inspector tab. On Firefox and Chromium based browsers this is done by pressing F12 and navigating to `Network`
2. Visit [Agama Lab](https://cloud.gluu.org/agama-lab) and login via Github
3. Attempt to select the repository. You should see activity in the network tab. Take a screenshot of the network tab and upload it here.

**Comment by nynymike** [comment]
Sometimes the cloud provider is spooling up Agama Lab, if it hasn't been very busy. So maybe just try again after a cup of coffee and see if it decides to cooperate...

**Reply by toroarnaud** [reply]
Thank you guys of the quick replies!
I've just tried with Firefox and ti works as expected! Sorry... I should have tried it earlier.
It still fails with Chrome and I attach the picture in case you want to have a look but since I can work with Firefox... No worries!
Thanks you again!
![Agmalab_chrome](https://github.com/JanssenProject/jans/assets/19341772/c1424693-3700-41c1-bf4e-ae3881095a86)

**Reply by SafinWasi** [reply, marked as answer]
No worries. By any chance, are you using an adblocker? Some of our backend calls get mistakenly blocked by adblockers. We do not serve any advertisements on Agama Lab, this is a known issue and we're working on fixing it.

**Comment by toroarnaud** [comment]
Indeed! Just checked it and that was my problem... (Again, I should have thought about it!)
Fixed! Thank you!!!
